pkg/infra/baseagent.go

```diff
@@ -135,11 +135,11 @@ func (a *Base) setHP(newHP int) {
 
 // Modeled as a first order system step answer (see documentation for more information)
 func (a *Base) updateHP(foodTaken food.FoodType) {
-	hpChange := a.tower.healthInfo.Width * (1 - math.Pow(math.E, -float64(foodTaken)/a.tower.healthInfo.Tau))
+	hpChange := int(a.tower.healthInfo.Width * (1 - math.Pow(math.E, -float64(foodTaken)/a.tower.healthInfo.Tau)))
```

Comment:
Why is this being set as an Int?

Reply:
Because the expression returns a float64, so needs to be cast to int.

Reply:
It's not needed as a float anywhere